README.md

```diff
@@ -18,46 +18,49 @@
 
 </div>
 
-***Shifu*** is a [Kubernetes](https://kubernetes.io) native framework designed to abstract out the complexity of interacting with IoT devices. ***Shifu*** aims to achieve TRUE plug-and-play IoT device virtualization, management, control, and automation.
+***Shifu*** is the next-generation open source IoT development framework, making developing an IoT app as simple as developing a Web app. The high extensibility of ***Shifu*** allows it to integrate almost all devices and protocols. Once integrating a device, ***Shifu*** generates a digital twin of the device as a microservice in the system and opens up device data and capabilities in the form of APIs. In this way, traditional IoT application development is reduced to simple web development, which greatly improves the efficiency, quality and reusability of IoT application development.
```

Comment:
`making developing an IoT app as simple as developing a Web app` - > `makes developing an IoT app as simple as developing a Web app`
`Once integrating a device` -> `Once a device is integrated`